src/test/java/com/aws/greengrass/logmanager/CloudWatchAttemptLogsProcessorTest.java

```diff
@@ -153,9 +165,13 @@ void GIVEN_one_user_component_one_file_less_than_max_WHEN_merge_THEN_reads_entir
                 .componentType(ComponentType.UserComponent)
                 .logFileInformationList(logFileInformationSet)
                 .build();
+
+        CloudWatchAttempt groupAttempt = new CloudWatchAttempt();
         logsProcessor = new CloudWatchAttemptLogsProcessor(mockDeviceConfiguration, defaultClock);
-        CloudWatchAttempt attempt = logsProcessor.processLogFiles(componentLogFileInformation);
-        assertNotNull(attempt);
+        groupAttempt.setLogGroupName(logsProcessor.getLogGroupName(componentLogFileInformation.getComponentType(),
+                "TestComponent"));
+        CloudWatchAttempt attempt = logsProcessor.processSingleLogFile(groupAttempt, componentLogFileInformation,
+                logFileInformationSet.get(0), chan);
 
         assertNotNull(attempt.getLogStreamsToLogEventsMap());
```

Comment:
Instead of getting the attempt here (since we won't return it from `processSingleLogFile`) in a later refactor assert that what we expect get added to the `groupAttempt`

Reply:
Get it